types/expirymanager/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for expirymanager 0.9
+// Project: https://github.com/SocketCluster/expirymanager
+// Definitions by: Daniel Rose <https://github.com/DanielRose>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type Key = any;
```

Comment:
What does this do?  Is this just a way of putting the name "Key" in scope?

Reply:
Oh, maybe it's expected to be filled in later?

Reply:
The usage is that you can place anything (as "key") into the ExpiryManager via expire(), with the time when those items should be expired. Those items can then be retrieved later via the various other methods.

To show where the key is used (even though it can be anything), I introduced the types Key and Keys. Or should I do this differently?

Reply:
I think using it to communicate intent is fine.